sdk/translation/azure-ai-translation-document/pom.xml

```diff
@@ -84,7 +84,7 @@ Code generated by Microsoft (R) TypeSpec Code Generator.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.12.2</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.12.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
Revert. Not sure why this should have happened.
@mssfang The 2 files were auto generated using the script: `
.\eng\common\scripts\Prepare-Release.ps1 azure-ai-translation-text translation MM/DD/YYYY

Reply:
Reverted